make

```diff
@@ -87,17 +113,17 @@ function images () {
   cp src/favicon.ico dist/favicon.ico
 
   while IFS= read -r -d '' FILE; do
-    CACHE_FILE="$CACHE_DIR/${FILE##"src/"}-$(md5 -q "$FILE")"
+    CACHE_FILE="$CACHE_DIR/${FILE##"src/"}-$(md5_hash "$FILE")"
     if ! [ -f "$CACHE_FILE" ]; then
       case ${FILE##*.} in
         "gif")
-          gifsicle --lossy --optimize=3 "$FILE" > "$CACHE_FILE"
+          gifsicle --optimize=3 "$FILE" > "$CACHE_FILE"
           ;;
         "jpg")
           jpegtran -progressive -optimize -copy none -outfile "$CACHE_FILE" "$FILE"
           ;;
         "png")
-          pngcrush -reduce -s -warn "$FILE" "$CACHE_FILE"
```

Comment:
For pngcrush I have 1.8.11, which has the `-warn` flag.  This was [added in 1.8.9](https://pmt.sourceforge.io/pngcrush/ChangeLog.html) it appears, so may just be a versioning issue?

Reply:
It is a versioning thing...now 2.0 is out, so I'm upgraded.

Reply:
Is there a fork?  I just checked homebrew and it's still `1.8.13`, not seeing a 2.0 on the project page anywhere either.